spotbugs/etc/messages.xml

```diff
@@ -6835,8 +6825,8 @@ Often, this indicates an error, because the value computed is never
 used.
 </p>
 <p>
-Note that Sun's javac compiler often generates dead stores for
-final local variables.  Because SpotBugs is a bytecode-based tool,
+Note that Oracle's javac compiler often generates dead stores for
+final local variables. Because SpotBugs is a bytecode-based tool,
```

Comment:
Personally not sure how current Oracle's javac compiler behaves. If it is working fine, the new description could be a kind of harmful rumors or misinformation.
Better to confirm it before we merge this change.

Reply:
You are right, I reverted this change